### PR TITLE
[fix] Vehicle expense report not working

### PR DESCRIPTION
**Pull request by rohitwaghchaure**

### CODE REVIEW

File: erpnext/hr/report/vehicle_expenses/vehicle_expenses.js [hunk 1]
```diff
@@ -22,18 +22,6 @@ frappe.require("assets/erpnext/js/financial_statements.js", function() {
 						query_report.trigger_refresh();
 					});
 				}
-			},
```
Comment (nabinhait) [inline erpnext/hr/report/vehicle_expenses/vehicle_expenses.js]: As you have removed from_date and to_date filters, there is no point of setting those filter values on change of fiscal _year.

File: erpnext/hr/report/vehicle_expenses/vehicle_expenses.py [hunk 2]
```diff
@@ -3,50 +3,47 @@
...
-	if filters.to_date:
-		conditions += " and date <= %(to_date)s"
+	fy = frappe.db.get_value('Fiscal Year', filters.get('fiscal_year'), ['year_start_date', 'year_end_date'], as_dict=True)
 	data = frappe.db.sql("""select vhcl.license_plate as "License",vhcl.make as "Make",vhcl.model as "Model",
-							vhcl.location as "Location",log.name as "Log",log.odometer as "Odometer",log.date as "Date",
-							log.fuel_qty as "Fuel Qty",log.price as "Fuel Price"
-							from `tabVehicle` vhcl,`tabVehicle Log` log
-							where vhcl.license_plate = log.license_plate and log.docstatus = 1 %s
-							order by date""" % (conditions,),filters, as_dict=1)
+		vhcl.location as "Location",log.name as "Log",log.odometer as "Odometer",log.date as "Date",
```
Comment (nabinhait) [inline erpnext/hr/report/vehicle_expenses/vehicle_expenses.py]: Use space after comma. In this case, use separate line for each column.

**Review by nabinhait — CHANGES_REQUESTED** [review]

File: erpnext/hr/report/vehicle_expenses/vehicle_expenses.js (hunk 1 above)
Comment (rohitwaghchaure) [inline erpnext/hr/report/vehicle_expenses/vehicle_expenses.js]: fy = frappe.db.get_value('Fiscal Year', filters.get('fiscal_year'), ['year_start_date', 'year_end_date'], as_dict=True)
Fetched start date and end date from the fiscal year

File: erpnext/hr/report/vehicle_expenses/vehicle_expenses.py (hunk 2 above)
Comment (rohitwaghchaure) [inline erpnext/hr/report/vehicle_expenses/vehicle_expenses.py]: Fixed

**Pull request merged**